release.yaml

```diff
@@ -8,12 +8,15 @@ ansible-roles:
   metal-roles:
     repository: https://github.com/metal-stack/metal-roles
     version: v0.8.5
+  riemers.gitlab-runner:
+    repository: https://github.com/riemers/ansible-gitlab-runner
+    version: v1.6.53
```

Comment:
This should not be an official dependency for metal-stack. Just install this role upfront your deployment.